src/track/track_gps_l2cm.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Jacob McNamee <jacob@swiftnav.com>
+ * Contact: Adel Mamin <adelm@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+#ifndef SWIFTNAV_TRACK_GPS_L2CM_H
+#define SWIFTNAV_TRACK_GPS_L2CM_H
+
+#include <libswiftnav/common.h>
+
+# define L2C_WEAK
```

Comment:
Let's remove this for now. L2C API weak attributes will be assigned in another PR.